pkg/downloader/store.go

```diff
@@ -29,6 +29,7 @@ type storeDownloader struct {
 	OCIDownloader
 	os, arch string
 	extFile  string
+	imagePrefix string
```

Comment:
The unit tests failed because this field is empty.

Reply:
You need to set the default value in function `NewStoreDownloader` to keep the back compitable.